### PR TITLE
feat: add ConicalGradient and fix slider

### PR DESCRIPTION
**Pull request by paruki-gin**
## Owner

- [ ] Nickname(Tuya):  若叶
- [ ] Intra-group review（组内审阅人员）：萧瑟

## Description

Add new component: ConicalGradient. fix bug of NumberSlider, CirclePicker and BrightRectSlider

## Package

- [ ] tuya-panel-animation-sdk
- [ ] tuya-panel-api
- [ ] tuya-panel-electrician-sdk
- [ ] tuya-panel-ipc-sdk
- [x] tuya-panel-lamp-sdk
- [ ] tuya-panel-lock-sdk
- [ ] tuya-panel-robot-sdk
- [ ] tuya-panel-outdoor-sdk

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Self CheckList

Please check all options.

- [x] Intra-group review
- [x] ESLint lint
- [x] Example Run
- [x] Jest Run
- [x] No Grammar Error
- [x] No Spell Error
- [x] No Merge Conflict

### CODE REVIEW

File: packages/tuya-panel-lamp-sdk/src/components/gradient/conical-gradient/Normal.tsx [hunk 1]
```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import _ from 'lodash';
+import Svg, { G, LinearGradient, Path, Defs, Stop } from 'react-native-svg';
+import { interpolateHcl as interpolateGradient } from 'd3-interpolate';
```
[the remark this answers is not in the document]
Comment (youngjuning) [reply packages/tuya-panel-lamp-sdk/src/components/gradient/conical-gradient/Normal.tsx:4]: 麻烦确认下这个问题！

[the remark this answers is not in the document]
Comment (paruki-gin) [reply packages/tuya-panel-lamp-sdk/src/components/gradient/conical-gradient/Normal.tsx:4]: add @types/d3-interpolate

File: packages/tuya-panel-lamp-sdk/src/components/gradient/conical-gradient/index.android.tsx [hunk 2]
```diff
@@ -0,0 +1,164 @@
+import React, { Component } from 'react';
+import { View } from 'react-native';
+import _ from 'lodash';
+import { Svg, Circle, Defs, Stop, G } from 'react-native-svg';
+import { interpolateHcl as interpolateGradient } from 'd3-interpolate';
```
[the remark this answers is not in the document]
Comment (paruki-gin) [reply packages/tuya-panel-lamp-sdk/src/components/gradient/conical-gradient/index.android.tsx:5]: add @types/d3-interpolate

**Review by youngjuning — COMMENTED** [review]
LGTM~

**Pull request merged**